src/algebra/module/submodule/basic.lean

```diff
@@ -257,6 +258,33 @@ lemma injective_subtype : injective p.subtype := subtype.coe_injective
 @[simp] lemma coe_sum (x : ι → p) (s : finset ι) : ↑(∑ i in s, x i) = ∑ i in s, (x i : M) :=
 map_sum p.subtype _ _
 
+/-! ### Actions by `submodule`s
+
+These instances tranfer the action by an element `m : M` of a `R`-module `M` written as `m +ᵥ a`
+onto the action by an element `s : S` of a submodule `S : submodule R M` such that
+`s +ᵥ a = (s : M) +ᵥ a`.
+
+These instances work particularly well in conjunction with `module.to_add_action`, enabling
+`s +ᵥ m` as an alias for `↑s + m`.
```

Comment:
`module.to_add_action` doesn't exist. Perhaps you meant to leave this as the original text?

Reply:
Yeah, I copy-pasted and changed `mul` to `add`... I'll just delete this part.